rust/openvasctl/src/openvas_redis.rs

```diff
@@ -28,16 +28,12 @@ where
     /// Initialize a RedisHelper struct with the connection to access the NVT cache
     /// and a empty task knowledge base to store the scan configuration to be sent to openvas.
     pub fn init(
```

Comment:
Since the initialization is done previously I think it would be better to rename it to new and remove the result.